scripts/build_all.py

```diff
@@ -0,0 +1,72 @@
+# language governing permissions and limitations under the License.
+import argparse
+import os
+import subprocess
+
+VERSION = '1.13.1'
+REPO = 'sagemaker-tensorflow-scriptmode'
+PY2_CPU_BINARY = 'https://s3-us-west-2.amazonaws.com/tensorflow-aws/1.13/AmazonLinux/cpu/latest-patch-latest-patch/tensorflow-1.13.1-cp27-cp27mu-linux_x86_64.whl'
+PY3_CPU_BINARY = 'https://s3-us-west-2.amazonaws.com/tensorflow-aws/1.13/AmazonLinux/cpu/latest-patch-latest-patch/tensorflow-1.13.1-cp36-cp36m-linux_x86_64.whl'
+PY2_GPU_BINARY = 'https://s3-us-west-2.amazonaws.com/tensorflow-aws/1.13/AmazonLinux/gpu/latest-patch-latest-patch/tensorflow-1.13.1-cp27-cp27mu-linux_x86_64.whl'
+PY3_GPU_BINARY = 'https://s3-us-west-2.amazonaws.com/tensorflow-aws/1.13/AmazonLinux/gpu/latest-patch-latest-patch/tensorflow-1.13.1-cp36-cp36m-linux_x86_64.whl'
+DEV_ACCOUNT = '142577830533'
+REGION = 'us-west-2'
+
+def _parse_args():
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--account', type=str, default=DEV_ACCOUNT)
+    parser.add_argument('--region', type=str, default=REGION)
+    parser.add_argument('--version', type=str, default=VERSION)
+    parser.add_argument('--py2-cpu-binary', type=str, default=PY2_CPU_BINARY)
+    parser.add_argument('--py3-cpu-binary', type=str, default=PY3_CPU_BINARY)
+    parser.add_argument('--py2-gpu-binary', type=str, default=PY2_GPU_BINARY)
+    parser.add_argument('--py3-gpu-binary', type=str, default=PY3_GPU_BINARY)
+    parser.add_argument('--repo', type=str, default=REPO)
+
+    return parser.parse_args()
+
+
+args = _parse_args()
+binaries = {
+    'py2-cpu': args.py2_cpu_binary,
+    'py3-cpu': args.py3_cpu_binary,
+    'py2-gpu': args.py2_gpu_binary,
+    'py3-gpu': args.py3_gpu_binary
+}
+build_dir = 'docker/{}'.format(args.version)
+
+# Run docker-login so we can pull the cached image
+login_cmd = subprocess.check_output(
+    'aws ecr get-login --no-include-email --registry-id {}'.format(args.account).split())
+print(login_cmd)
+subprocess.check_call(login_cmd.split())
+
+for arch in ['cpu', 'gpu']:
+    for py_version in ['2', '3']:
+        binary_url = binaries['py{}-{}'.format(py_version, arch)]
+        binary_file = os.path.basename(binary_url)
+        cmd = 'wget -O {}/{} {}'.format(build_dir, binary_file, binary_url)
+        print(cmd)
+        subprocess.check_call(cmd.split())
+        tag = '{}-{}-py{}'.format(args.version, arch, py_version)
+        prev_image_uri = '{}.dkr.ecr.{}.amazonaws.com/{}:{}'.format(args.account, args.region, args.repo, tag)
+        build_cmd = 'docker build -f {}/Dockerfile.{} --cache-from {} ' \
+                    '--build-arg py_version={} --build-arg framework_installable={} ' \
+                    '-t {}:{} {}'.format(build_dir, arch, prev_image_uri, py_version, binary_file, args.repo, tag, build_dir)
+        print(build_cmd)
+        subprocess.check_call(build_cmd.split())
+        print('Deleting binary file')
```

Comment:
I'd add the binary file name to this print statement

Reply:
k